File: src/main/java/me/anviks/codewars/_6kyu/multiples_of_3_or_5/SolutionTest.java
package me.anviks.codewars._6kyu.multiples_of_3_or_5;

/*
 * https://www.codewars.com/kata/514b92a657cdc65150000006
 */

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class SolutionTest {
    @Test
    public void test() {
        assertEquals(23, new Solution().solution(10));
    }
}

File: kotlin/src/main/kotlin/me/anviks/codewars/solutions/_6kyu/playing_with_passphrases/SolutionTest.kt
/*
 * https://www.codewars.com/kata/559536379512a64472000053
 */

package me.anviks.codewars.solutions._6kyu.playing_with_passphrases

import org.junit.Test
import kotlin.test.assertEquals

class SolutionTest {

    @Test
    fun basicTests() {
        assertEquals("!!!VpZ FwPm j ", Solution.playPass(" I LOVE YOU!!!", 1))
        assertEquals("!!!uOy eVoL I", Solution.playPass("I LOVE YOU!!!", 0))
        assertEquals("zDdCcBbB", Solution.playPass("AAABBCCY", 1))
    }
}
